src/frontend/templates/pages/games_list.html

```diff
@@ -0,0 +1,152 @@
+{% extends 'base.html' %}
+
+{% block title %}
+Games for {{network.name}}
+{% endblock %}
+
+{% block content %}
+
+{% if kind == "training" %}
+<h1 class="title">Training Games for {{network.name}}</h1>
+{% else %}
+<h1 class="title">Rating Games for {{network.name}}</h1>
+{% endif %}
+
+{% if kind == "training" %}
+<div class="pagination">
+  <span class="step-links">
+    {% if page_obj.has_previous %}
+    <a href="{% url 'training_games_list' run.name network.name 1 %}">&laquo; First</a>
+    <a href="{% url 'training_games_list' run.name network.name page_obj.previous_page_number %}">Prev</a>
+    {% else %}
+    &laquo; First
```

Comment:
We can use font awesome icons instead of &laquo;

Eg `<i class="fab fa-github"></i>` will display github icon

Reply:
https://fontawesome.com/icons?d=gallery

Reply:
Ooh nice icons. Okay done!

Reply:
nicee